platforms/gke/base/core/container_cluster/container_cluster.tf

```diff
@@ -200,33 +171,13 @@ resource "google_container_cluster" "cluster" {
 
   node_pool {
     initial_node_count = 1
-    name               = "system"
-
-    autoscaling {
-      location_policy      = "BALANCED"
-      total_max_node_count = 32
-      total_min_node_count = 1
-    }
-
-    network_config {
-      enable_private_nodes = true
-    }
-
+    name               = "default-pool"
     node_config {
-      machine_type    = "e2-standard-4"
+      machine_type    = var.cluster_system_node_pool_machine_type
```

Comment:
Just checking: we're reusing the same variable as we're using in the `system` node pool below.